### ISSUE TITLE
No bounding box or a lot of boxes(false positive)

### ISSUE THREAD

**Issue by 0merjavaid** [thtrieu/darkflow#363]
I'm training from scratch on VOC2007, 20 classes using yolo-voc. 
batch size=5
lr=1e-5
steps reached 10000+
average loss =7-10 fluctuating 
Even after i saved model with minimum loss 2.9 on 9375th step with above stats.
but no bounding box with threshold even at 0.05? 
![screenshot from 2017-07-31 02-05-49](https://user-images.githubusercontent.com/22716318/28757075-d19e93b8-7594-11e7-88a9-a8c9503dc558.png)

and testing with 
flow --model cfg/yolo-voc.cfg --backup ckpt/ --load -1 --imgdir sample_img/ --threshold 0.05

why????

**Comment by Cpruce** [comment]
Same. I have to set the minimum confidence threshold to 0.0001 to see any results. loss plateaus after 50k steps (1-2 epochs) for custom dataset

**Comment by Zumbalamambo** [comment]
same

**Comment by onurbarut** [comment]
similar issue, but in my case all the bounding boxes are on the right edge of the image, all are in the same size, all the same for all of six of my training data.

**Comment by Zumbalamambo** [comment]
SSD is better than Yolo.. I fixed my issue with SSD. Go for it mate..

**Comment by ProtossDragoon** [comment]
Anyone solved this problem..?
